### DISCUSSION TITLE
contributing custom nodes

### DISCUSSION THREAD

**Discussion by felixniemeyer** [cables-gl/cables_docs#634] [Answers & Questions] [answered]
Hi, I'm working on a real-time audio analysis tool. 
I'm thinking about creating it as a node in cables. 

What would be the recommended way to do so?
- fork
- check out cables-gl locally
- propose custom node as a pull request?

Thanks everyone

**Comment by felixniemeyer** [comment]
Oh I just realized cables isn't open source, right?

**Reply by pandrr** [reply]
not yet! we are working on it

**Comment by pandrr** [comment, marked as answer]
the best way would be to create a team, make it public and let us and others try your ops!
(rename your ops into the team namespace)

**Reply by steam0r** [reply]
update the docs accordingly: https://cables.gl/docs/faq/features/contribute/contribute

**Comment by felixniemeyer** [comment]
Awesome, thanks everyone. 
I would like to be able to show you something next year.
